examples/IBFE/explicit/ex4/example.cpp

```diff
@@ -30,7 +30,7 @@
 #include <libmesh/boundary_info.h>
 #include <libmesh/equation_systems.h>
 #include <libmesh/exodusII_io.h>
-#include <libmesh/mesh.h>
+#include <libmesh/replicated_mesh.h>
```

Comment:
Can you move this to the end of the list of libMesh includes (I think that this is what `clang-format` should do for this particular change, to keep the `#include` directives in alphabetical order).